src/pcre2_compile_class.c

```diff
@@ -44,6 +44,30 @@ POSSIBILITY OF SUCH DAMAGE.
 
 #include "pcre2_compile.h"
 
+/* Checks the allowed tokens at the end of a class structure in debug mode.
+When a new token is not processed by all loops, and the token is equals to
+a) one of the cases here:
+   the compiler will complain about a duplicated case value.
+b) none of the cases here:
+   the loop without the handler will stop with an assertion failure. */
+
+#ifdef PCRE2_DEBUG
+#define CLASS_END_CASES(meta) \
+  default: \
+  PCRE2_ASSERT((meta) <= META_END); \
```

Comment:
I'm surprised the compiler lets you write a statement that falls through to another case, with the "Falls through" comment that's enforced elsewhere.

Reply:
I forgot it. Thank you for noticing it. Compiler can complain about it with the suitable compiler options, probably something is missing.

Reply:
I guess the "multi-config" CI build doesn't include a debug build, with those options.